packages/client/src/scripts/please-login.ts

```diff
@@ -3,7 +3,7 @@ import { $i } from '@/account';
 import { i18n } from '@/i18n';
 import { popup } from '@/os';
 
-export function pleaseLogin(path?: string) {
+export function pleaseLogin(path?: string, stopExecution: boolean = true) {
```

Comment:
I think these two are exclusive: If a path is provided, the execution should continue (with the redirection), and if no path is provided,  the execution should not continue. So I think instead of adding a separate parameter you could check if the existing parameter is set or not.